experimental/uefi/proto/build.rs

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
It probalby doesn't matter a lot, but we normally split the code generation so that the server code is generated in the server crate and the client code in the client crate.

Reply:
Ah, we even have an utility to do that!

What I wanted to achieve here was to share a single `.proto` file between both client and server, and having a separate crate seemed the most natural thing to do. But as we have `oak_utils::generate_grpc_code` that deals exactly in this, I was able to drop the crate altogether.